bindings/c/CMakeLists.txt

```diff
@@ -334,6 +334,14 @@ endif()
             --process-number 3
           )
 
+    add_test(NAME fdb_c_upgrade_multi_threaded_710api
+        COMMAND ${CMAKE_SOURCE_DIR}/tests/TestRunner/upgrade_test.py
+            --build-dir ${CMAKE_BINARY_DIR}
+            --test-file ${CMAKE_SOURCE_DIR}/bindings/c/test/apitester/tests/upgrade/MixedApiWorkloadMultiThr.toml
+            --upgrade-path "7.1.3" "7.2.0" "7.1.3"
```

Comment:
duplicate entry. typo?

Reply:
unless this is the syntax for immediate upgrade then downgrade. in which case please resolve this comment

Reply:
Yes, the goal is to test an upgrade followed by a downgrade